docker/dockerfiles/Dockerfile.onnx.gpu

```diff
@@ -48,6 +50,7 @@ RUN pip3 install \
     --upgrade \
     && rm -rf ~/.cache/pip
 
+RUN pip3 install flash-attn --no-build-isolation
 FROM scratch
 COPY --from=base / /
```

Comment:
To capture: this change of base image blows up the docker image size but we're going to fast follow to only copy the things we need from the build image in to the distributed image.

Reply:
this change? not the base image change at the top of the dockerfile?